src/Microsoft.Tye.Hosting/Model/DockerRunInfo.cs

```diff
@@ -13,10 +13,11 @@ public DockerRunInfo(string image, string? args)
         {
             Image = image;
             Args = args;
+            IsAspNet = Image.StartsWith("mcr.microsoft.com/dotnet/core/aspnet", StringComparison.OrdinalIgnoreCase);
```

Comment:
Is there a point to still having this initializer?  This is the name of the base-image. Someone using a pre-built image as a dependency would have this base-image as a layer (possibly), not as the name of their image. I'm struggling to think of a case where this would be true (other than when we transform a project).

If we want to go deep on lighting up asp.net core functionality for arbitrary images, then a label is probably the right way to do is.

Reply:
I don't want to add IsAspNet to ContainerServiceBuilder. Here's where I'm using it here https://github.com/dotnet/tye/blob/122d73c3e55135ffdf85f1e52e222b17e733aab6/test/E2ETest/TyeRunTests.cs#L212.

Do you want to support container labels as part of this PR?